spark-board-ui/src/App.jsx

```diff
@@ -78,7 +75,22 @@ export default function App() {
             // orgnaize the nodes
             organizeNodes();
         }
-    }, [nodesInitialized]);
+    }, [nodesInitialized, settings]);
```

Comment:
Won't this make the nodes reorder when the settings are updated?

Reply:
Yes, I liked it when testing, but I will revert it